README.md

```diff
@@ -1,36 +1,38 @@
 # Nerpa
 
-NRP Matcher is a tool which links gene cluster to known natural products.
+Nerpa is a tool for linking biosynthetic gene clusters (BGCs) to known nonribosomal peptides (NRPs).
+BGCs are predicted in genome sequences (FASTA or GBK) with [antiSMASH](https://antismash.secondarymetabolites.org/). 
+Known NRPs are accepted in the [SMILES format](https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system)
+ and processed with [rBAN](https://web.expasy.org/rban/).
 
 ## Dependencies
 
-NRP Matcher requires a 64-bit Linux system or MAC OS and Python 3,
-g++ (version 5.2 or higher) and cmake (version 3.5 or higher) to be
-pre-installed on it.
-
-Also https://github.com/ablab/dereplicator/ must be
-added to PATH. Folders "Fragmentation_rule" and "configs" must be
-located in one of the paths:
-- \<DEREPLICATOR INSTALL DIR\>/
-- \<DEREPLICATOR INSTALL DIR\>/../
-- \<DEREPLICATOR INSTALL DIR\>/../../
-- \<DEREPLICATOR INSTALL DIR\>/../share/
-- \<DEREPLICATOR INSTALL DIR\>/../share/npdtools/
+### Required
+* 64-bit Linux system or macOS
+* g++ v.5.2+ or clang v.3.2+
+* cmake v.3.5+
+* Python v.3.6+
+* Python libraries:
+    * [RDKit](https://www.rdkit.org/docs/Install.html)
+    * [networkx](https://networkx.org/documentation/stable/install.html)
 
+### Conditionally required
+The following dependencies are required only for specific input data types.
+* java (only if NRPs are provided as SMILES)
```

Comment:
or as rBAN output json; we do not run rBAN only if we provide Nerpa already linearized structures (this mode is also not implemented yet)

Reply:
Actually, the only way not to run rBAN now is to provide structures in the custom graph format (not documented yet). So, this section was mostly added as a stub for future (real) "Conditional requirements", e.g. `antiSMASH` in `PATH` to enable FASTA/GBK input (not implemented yet :) )